### DISCUSSION TITLE
Basic API form - example

### DISCUSSION THREAD

**Discussion by IanMuller** [YetiForceCompany/YetiForceCRM#16508] [How to]
Ladies and gentleman, is is possible to have an example of code what can be reused to build online form? 
Maybe some link?
Struggling with understanding things I can find here :/

**Comment by mariuszkrzaczkowski** [comment]
it will be useful to you:
https://github.com/YetiForceCompany/YetiForceScripts/tree/master/YetiREST
someone has written about it before, look for it

**Comment by IanMuller** [comment]
Thank you. I saw that... searched too, but still have no steps on how to configure API and call it from for instance simple html form :-)

**Reply by mariuszkrzaczkowski** [reply]
https://github.com/YetiForceCompany/YetiForceCRM/discussions/13932
https://github.com/YetiForceCompany/YetiForceCRM/discussions/14393
https://github.com/YetiForceCompany/YetiForceCRM/issues/13740

**Comment by apcloic** [comment]
@IanMuller 
Also check this one regarding Google forms to Yetiforce using Google Sheets API 
#13933

**Comment by JohnMaverick** [comment]
I think maybe it will be good idea to provide community with straight-forward sample:

HTML FORM  > FILE WITH CONFIGURATION TRIGERRED BY FORM > API SET OF FILES

Something very basic but easy to digest for beginners. Such sample to be included in "scripts" in GIT repo here

**Reply by apcloic** [reply]
@JohnMaverick 
I've just made one simple example with an HTML basic form.
https://github.com/apcloic/htmlform2yetiforce
Hope it helps.
Regards,
